src/scripts-upload-media/factcheck.js

```diff
@@ -0,0 +1,97 @@
+const axios = require("axios");
+const path = require("path");
+const { v4: uuid } = require("uuid");
+require("dotenv").config({
+    path: path.join(__dirname, "development.env"),
+});
+const fs = require("fs")
+
+const accessToken = process.env.ACCESS_TOKEN;
+const datasource = "60212d4c-e870-46a6-907b-88ef39e0b88f";
+const knownMediaTypes = ['text', 'image', 'video'];
+const ignoreMediaTypes = ['tweet', 'facebook', 'instagram'];
+const failedRequestsFile = "factcheck-failed-requests.json";
+
+const insertPosts = async () => {
+    const { connect, get, bulkWrite } = require("../backend/core/mongo/index")
+    await connect()
+
+    const stories = await get(
+        "kosh_metadata",
```

Comment:
Lets make this a const variable at the top of the file. We can name it "FACTCHECK_DB_NAME". I believe this is not the right name of that collection in our production database. so having it at the top will make it easy to change later